src/librustc_span/symbol.rs

```diff
@@ -26,12 +26,14 @@ symbols! {
     Keywords {
         // Special reserved identifiers used internally for elided lifetimes,
         // unnamed method parameters, crate root module, error recovery etc.
+        fn is_special:
```

Comment:
This confuses code highlighting in my editor (vscode + rust-analyzer), but `fn` is required for searchability.
Perhaps the list needs to go into braces, e.g. like this:
```rust
fn is_special {
    Invalid:            "",
    PathRoot:           "{{root}}",
    DollarCrate:        "$crate",
    Underscore:         "_",
}
```